### PR TITLE
chore: tidy image component

### PR DESCRIPTION
**Pull request by parkjh**
- no functionality has changed at all
- notes that all the snapshot remains the same
- there are a lot of components within Image, which begs the question whether they should be a package of its own?

### CODE REVIEW

File: packages/image/modal-image.showcase.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/no-danger, react/prop-types */
 import React, { Fragment } from "react";
 import { Text, View } from "react-native";
-import ModalImage from "./src/modal-image";
+import ModalImage from "./src/modalImage";
```
Comment (jmars) [inline packages/image/modal-image.showcase.js]: We usually use snake-case

File: packages/image/src/safeAreaView/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import SafeArea from "./react-native-safe-area";
+
+export default SafeArea;
```
Comment (jmars) [inline packages/image/src/safeAreaView/index.js]: Is this necessary, just import react-native-safe-area where you use it?

File: packages/image/src/utils/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import appendURL from "./appendToURL";
+
+export default appendURL;
```
Comment (jmars) [inline packages/image/src/utils/index.js]: ?

File: packages/image/src/image.js [hunk 4]
```diff
@@ -8,10 +8,10 @@ import {
   normaliseWidthForAssetRequestCache,
   convertToPixels
 } from "@times-components/utils";
-import LazyLoadingImage from "./lazy-loading-image";
+import LazyLoadingImage from "./lazy-loading-image/lazy-loading-image";
 import appendToUrl from "./utils";
 import { defaultProps, propTypes } from "./image-prop-types";
-import Placeholder from "./placeholder";
+import Placeholder from "./placeholder/placeholder";
```
Comment (ValGeorgiev) [inline packages/image/src/image.js]: you can use only ./placeholder because you have index.js file

File: packages/image/src/image.web.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import React, { Component } from "react";
 import { View } from "react-native";
 import { addMissingProtocol } from "@times-components/utils";
-import Placeholder from "./placeholder";
+import Placeholder from "./placeholder/placeholder";
```
Comment (ValGeorgiev) [inline packages/image/src/image.web.js]: same ^

**Pull request merged**